omniduct/databases/sqlalchemy.py

```diff
@@ -87,7 +87,7 @@ def _dataframe_to_table(self, df, table, if_exists='fail', **kwargs):
                          index=False, if_exists=if_exists, **kwargs)
 
     def _cursor_empty(self, cursor):
-        return False
+        return cursor is None
```

Comment:
perhaps this should be the defintion in base?

Reply:
Yep. I think that is probably a good idea :).